frontend/apps/case-exploration/src/feature/cells/priority-cell.tsx

```diff
@@ -0,0 +1,32 @@
+import { Indicator, IndicatorProps } from '@/components/base/ui/indicator';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/base/ui/tooltip';
+import { useI18n } from '@/components/hooks/i18n';
+
+type PriorityCellProps = {
+  code: 'asap' | 'routine' | 'stat' | 'urgent';
+};
+
+const colors: Record<string, IndicatorProps['variant']> = {
+  asap: 'amber',
+  routine: 'slate',
+  stat: 'red',
+  urgent: 'blue',
+};
+
```

Comment:
if we hardcode colors like this, it would not be possible in the future to adapt to new portals easily, it should be themable.